### DISCUSSION TITLE
Withdraw and Wallet hand-off fees

### DISCUSSION THREAD

**Discussion by jcnelson** [stacks-archive/sbtc-developer-release#325] [Ideas]
One aspect of the sBTC system that may surprise users is the fact that the Stackers need to pay a BTC fee each time they perform a wallet hand-off.  That extra BTC has to come from somewhere, but who pays it?

The SIP currently calls for _every sBTC holder_ to pay the fee, collectively, in the form of a _pro rata_ deduction in their sBTC holdings.  This would ensure that the total outstanding sBTC gets reset to whatever the new BTC balance is, minus the hand-off fee.  However, users may find this surprising (and negative) because their sBTC holdings diminish simply by keeping the sBTC in the system.  The BTC fee is a "decentralization fee" -- it's the cost of having a rotating set of signers (Stackers), which necessitates re-keying the BTC to the new signers, which in Bitcoin's case, necessitates converting all the old wallet UTXOs into new wallet UTXOs on the hand-off.  Moreover, this fee is a function of the total number of UTXOs, which can be large and unpredictable.

What are some alternatives?

**Comment by AshtonStephens** [comment]
Closing as duplicate of https://github.com/stacks-network/sbtc/discussions/323
